index.html

```diff
@@ -7,29 +7,31 @@
 </head>
 <body>
 
-<h1>Client Test Pages</h1>
+<h1>Private Test Pages</h1>
 
 <p>This site contains a collection of pages with known web content (e.g. trackers, web functionality, etc) that can be used for testing by the extensions and mobile apps.<p>
 
 <h2>Trackers</h2>
 
 <ul>
-	<li><a href="./trackers/1major-via-script.html">1 major tracker loaded via script</a>
-	<li><a href="./trackers/1major-via-img.html">1 major tracker loaded via img</a>
-	<li><a href="./trackers/document-fragment.html">Image loaded via document fragment</a>
-
+	<li><a href="./trackers/1major-via-script.html">1 major tracker loaded via script</a></li>
+	<li><a href="./trackers/1major-with-surrogate.html">1 major tracker with surrogate</a></li>
+	<li><a href="./trackers/1major-via-img.html">1 major tracker loaded via img</a></li>
+	<li><a href="./trackers/document-fragment.html">Image loaded via document fragment</a></li>
 </ul>
 
 <h2>Browser Features</h2>
 
 <ul>
-	<li><a href="./features/geolocation.html">Geolocation</a>
+	<li><a href="./features/geolocation.html">Geolocation</a></li>
+	<li><a href="./features/target-blank.html">Links Open in New Window</a>
+    <li><a href="./features/closeable.html">window.close() Link</a>
```

Comment:
👀 I wish there was one more space there…

Reply:
this is why the internet is running out of space!